### ISSUE TITLE
Possibility to filter namespaces

### ISSUE THREAD

**Issue by bastienbc** [flux-web/flux-web#24]
Hi,

Could it be possible to limit the namespaces the flux-web display ?
The helm-operator I set is connected to tiller which is limited to only some namespaces.

So there is no point to display all the namespaces.
Could it be possible to whitelist namespaces at least ? In the ClusterRole maybe...

**Comment by idobry** [comment]
Yes, flux-web's chart creating a clusterrole https://github.com/flux-web/flux-web/blob/master/chart/flux-web/templates/rbac-clusterrole.yaml
So it's possible to change it as you want.

**Comment by idobry** [comment]
Any update? Can I close this?

**Comment by bastienbc** [comment]
Sorry I was on vacation.

Well this is not a normal way to use a helm chart. If I follow your reasoning to the end, there is not any good reason to fix anything on the helm chart, because I can just download it and change it.
Or I could just fix it AFTER that helm has done its job.

But if I download it and manage it my way, I will not have any improvement automatically, and if I change what helm deploys, why would I use helm?

Am I wrong?
